Frontend/app/src/main/java/io/happybugs/happybugs/MainActivity.java

```diff
@@ -20,18 +22,13 @@
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+
+        BottomNavigationView bottomNavigationView = (BottomNavigationView)
```

Comment:
There's no strict rule, but when we do line-wrapping, we usually break after =. Different teams have different preference, but I most of my teams followed this convention. Just FYI.

Reply:
Thank you for your advice! changed line-wrapping.